howso/return_types.amlg

```diff
@@ -890,6 +890,25 @@
 					}
 				}
 			}
+		CombineResponse
+			{
+				type "assoc"
+				additional_indices (false)
+				indices {
+					"status" {
+						type ["string" "null"]
+						enum ["analyze" "analyzed"]
```

Comment:
Want to double check we don't need `reduce_data` here as well (looks like its missing from TrainResponse/ProcessTrainPayloadResponse so would be good to add it there)